docs/lambda.asciidoc

```diff
@@ -60,6 +59,8 @@ exports.handler = apm.lambda(function handler (payload, context, callback) {
 
 The agent will now monitor the performance of your lambda function.
 
+*IMPORTANT*: During installation of the Lambda extension you'll have set a number of environmental variables for configuring the agent.  We recommend relying on these environmental variables -- while it's possible to set configuration values via the configuration object passed to `start`, doing so may interfere with the configuration required by the Lambda extension.
```

Comment:
nit: I think "environment variable" is the more common terminology than "environmental variable" and is the term used in the current docs.  One data point: https://trends.google.com/trends/explore?geo=CA&q=%22environment%20variable%22,%22environmental%20variable%22

Reply:
++